src/mappers/db_mapper.js

```diff
@@ -0,0 +1,22 @@
+class DbMapper {
+  constructor(db) { this.db = db }
+
+  findAll() {
+    return this.db.collection('Courses').find() // [Course, Course, Course, ...]
+  }
+
+  insert(courses) {
+    return this.db.collection('Courses').insert(courses.toJSON())
+  }
+  update(content, course) {
+    return this.db.collection('Courses').update(content.toJSON(), course)
+  }
+  getUsers() {
+    return this.db.collection('Users').getUsers() // [User, User, User, ...]
+  }
+  createUser(user, users) {
+    return this.db.collection('Users').createUser(user, users)
```

Comment:
Nas funções declaradas dentro da classe DbMapper, no método construtor (linha 2) já definimos com o "this.db", qual db deve ser recebido corretamente, é necessário utilizar o "this." novamente, dentro das outras funções?